.github/workflows/l10n.yml

```diff
@@ -0,0 +1,27 @@
+name: daily l10n run
+
+on:
+  schedule:
+    - cron: "0 0 * * 1-5"
```

Comment:
Can you add a comment with what this cron means? I almost always have to look it up!